README.md

```diff
@@ -68,6 +68,9 @@ See [Wrappers](WRAPPERS.md)
 
 
 ## Release and branch model
+
```

Comment:
These changes need to be removed from the PR, maybe a rebase against current dev branch helps.